src/fal/cli/dbt_runner.py

```diff
@@ -59,7 +59,7 @@ def dbt_run(args):
         command_list += ["--profiles-dir", args.profiles_dir]
 
     if args.select:
-        command_list += ["--select"] + args.select
+        command_list += ["--select"] + models_list
     if args.exclude:
         command_list += ["--exclude"] + args.exclude
```

Comment:
Should we remove `--exclude` and others selectors? I think these are no longer necessary

Reply:
exclude will be necessary right, but the logic behind is not implemented yet.. or are you saying we shouldnt support `--exclude` at all?

Reply:
What I'm saying is the select, exclude logic is no longer done in dbt, we do it. So we just pass the `--select` with the correct values. Right?

Let's say:

```
fal flow run --exclude b --select a+
```

With the given nodes:

```mermaid
  graph LR;
      a-->b;
      a-->c;
      b-->d;
      c-->e;
      c-->d;
  style a fill:green
  style b fill:red
  style c fill:green
  style d fill:green
  style e fill:green
```

You figure out that you don't have to do `b` at all in fal, and then you pass to dbt:

```
dbt run --select a c d e
```

which gives a correct run. 

Notice that the `--exclude` is not necessary for the dbt invocation to do the correct thing.

Reply:
yea I see what you mean, I was thinking we can pass exclude if its just exclude.

 fal flow run --exclude b 
 
 but even then we probably will calculate the dbt models..